app/controllers/chefs_controller.rb

```diff
@@ -0,0 +1,11 @@
+class ChefsController < ApplicationController
+
+  def show
+    @chef = Chef.find(params[:id])
+  end
+
+  def ingredients
+    @chef = Chef.find(params[:id])
+    @ingredients = @chef.ingredients
+  end
```

Comment:
If I understand correctly, this is for displaying all of the chef's ingredients for the ingredients index page, so my question is why we are routing it through the chef's controller and not on the `index` action of the `IngredientsController`?

This is what your route looks like on the chef show page: `"/chefs/#{@chef.id}/ingredients"` so I would have definitely routed it to the `IngredientsController` and did something like this:

```ruby
class IngredientsController < ApplicationController
  def index
    @chef = Chef.find(params[:id])
    @ingredients = @chef.ingredients
  end
end
```

This is exactly what you did in your `Chefs::IngredientsController`.